### DISCUSSION TITLE
is there a guide to code structure?

### DISCUSSION THREAD

**Discussion by duo1204** [multitheftauto/mtasa-blue#1968] [Q&A] [answered]
i'm first time contributing to any open source project.
does it work as if i have to read all these lines of code and understand structure in order to code
or is there a written guide to what class and function works how
(i found directory guide but that hardly helps to understand how it works)

thank you

**Comment by Disinterpreter** [comment, marked as answer]
Yes, it is. But at the moment the documentation a little short than it need.
Check out here:
https://github.com/multitheftauto/mtasa-blue/wiki/Style-Guide
https://github.com/multitheftauto/mtasa-blue/wiki

Other things would be described on code review.
